src/RichText.js

```diff
@@ -0,0 +1,82 @@
+import {Editor, ContentState, convertFromHTML, DefaultDraftBlockRenderMap, DefaultDraftInlineStyle, CompositeDecorator} from 'draft-js';
+const ReactDOM = require('react-dom');
+var sdk = require('./index');
+
+const BLOCK_RENDER_MAP = DefaultDraftBlockRenderMap.set('unstyled', {
+    element: 'p' // draft uses <div> by default which we don't really like
+});
+
+const styles = {
+    BOLD: 'strong',
+    CODE: 'code',
+    ITALIC: 'em',
+    STRIKETHROUGH: 's',
+    UNDERLINE: 'u'
+};
+
+export function contentStateToHTML(contentState:ContentState): String {
+    const elem = contentState.getBlockMap().map((block) => {
+        const elem = BLOCK_RENDER_MAP.get(block.getType()).element;
+        const content = [];
+        block.findStyleRanges(() => true, (s, e) => {
+            const tags = block.getInlineStyleAt(s).map(style => styles[style]);
+            const open = tags.map(tag => `<${tag}>`).join('');
+            const close = tags.map(tag => `</${tag}>`).reverse().join('');
+            content.push(`${open}${block.getText().substring(s, e)}${close}`);
```

Comment:
Some comments to explain what is going on here would be nice:
 - Why are we returning `true` in the first function to `findStyleRanges`?
 - What are `s` and `e`?
 - Why are we joining a list of open tags to look like `<a><b><c><d>`?
 - Why are we reversing the close tags?
 - Are `block.getText().substring(s, e)` guaranteed safe indexes?